### PR TITLE
remove podman stack

### PR DESCRIPTION
**Pull request by johrstrom**
I'm going to remove the podman stack becuase really we don't need it.

Essentially the use case for me was to map myself to a user inside the container.  After getting a lot of mileage out the podman images I found that I could do the same thing with the Docker image we simply enable these 2 things inside the container.

The idea is this:
$(id -u) (whoever you are) starts the image and maps to ood (uid 1000 inside the container).  So ood needs sudo privileges to start httpd. 

I had to remove the passwords because of tty problems, though I'm looking for a way around that now. But mounting the /etc/sudoers.d/ood file as $(id -u) is giving me problems.

### CODE REVIEW

**Comment by johrstrom** [comment]
I don't really need this anymore and really, it's probably confusing to have two container stacks.

**Comment by treydock** [comment]
FWIW if you need to do UID changing , look at build.sh in ondemand-packaging repo. I stole some wrapper ideas from passenger RPM build workflow that changes the `ood` user UID/GID inside the container to match UID/GID of person running docker so that files written in the container to a RW volume are accessible outside the container.

File: common/ood-setup.sh [hunk 1]
```diff
@@ -11,7 +11,9 @@ fi
 # Add user to system and apache basic auth
 groupadd ood
 useradd --create-home --gid ood ood
-echo -n "ood" | passwd --stdin ood
+passwd --delete root
+passwd --delete ood
+usermod -a -G wheel ood
```
Comment (treydock) [inline common/ood-setup.sh]: Why these changes?

Comment (johrstrom) [reply common/ood-setup.sh]: Got rid of removing root's password. Still trying to figure out Git reviews, so I removed the comments from before, but essentially the ask for a nil password and ood being a part of the weel group is this:  

ood (uid 1000) is the uid that starts the container. But it needs to be root for all sorts or things like booting httpd and interacting with all the httpd related file systems, and actually forking process' in the nginx stage. So ood needs to be wheel so that it can do all of that. I remove the password so it can freely do all of that in scripts. 

I do all of this so that I can be uid 1000 inside and out of the container. 

Again, I'm fine to just remove the podman stack and keep this file as is. It is used in packer image creation, where this would be problematic, so maybe this change would have to be coupled with (deferred to) changing how packer image gets created?

Comment (treydock) [reply common/ood-setup.sh]: This script is also used by docker image builds and vagrant too so I'd prefer to leave it as-is and if the changes are needed for podman we either drop podman or do something specific to podman outside common.

**Comment by johrstrom** [comment]
There's no need for this given the rake tasks that now build containers.
